09_employment/compute_metrics_wage_ratio_2022_subgroup_new.do

```diff
@@ -0,0 +1,200 @@
+/***************************
+This file imports the 2022 QCEW data and exports average weekly wage for each
+county. The file can be edited to read in data from any other year.
+
+Before using this file, download the relevant year of data from QCEW NAICS-Based Data Files, County High-Level (and select the annual summary)
+
+Programmed by Kevin Werner + updated by Kassandra Martinchek in 2023, adding subgroups
+
+2/12/2024
+
+Living wage data is in 2022 dollars, so it is deflated. 
+****************************/
+
+local raw "C:\Users\KMartinchek\Documents\umf-2024\mobility-from-poverty\09_employment"
+local wages "C:\Users\KMartinchek\Documents\umf-2024\mobility-from-poverty\09_employment"
+
+/***** save living wage as .dta *****/
+cd `wages'
+import delimited using "mit-living-wage-2022.csv", clear // while in code review, please note that I did not change this, as updated data required a fee and there was a no-scraping notice
```

Comment:
reminder to remove comment from internal review

Reply:
Removed